index.html

```diff
@@ -30,56 +30,84 @@
+          </g></svg>
+          </span>
+          <h4 class="main-small-heading">Dogs</h4>
+      </div>
+      <div class="dogs-cats-birds-rabbits-box dogs-cats-box">
+          <span class="u-icon u-icon-circle u-icon-2">
```

Comment:
Te klasy w spanie są niepotrzebne. Jak kopiujesz coś to staraj się zawsze sprzątać syf. W tamtym projekcie te klasy pewnie miały sens i coś robiły. Tutaj nie robią nic.

Reply:
Poprawione